dev-python/meson-python/meson_python-0.13.1.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="Short description of ProjectX"
```

Comment:
This looks wrong ;-)

Reply:
Yeah, was a bit too fast on this one I guess (same for the DESCRIPTION), will have a look again later. Thanks!